scalyr_agent/builtin_monitors/tests/docker_monitor_test.py

```diff
@@ -63,93 +64,95 @@ def fake_init(self):
-            def augment_user_agent(fragments):
-                counter['callback_invocations'] += 1
-                detected_fragment_changes.append(fragments[0])
-
-            # Decorate the get_user_agent_fragment() function as follows:
-            # Each invocation increments the FakeClockCounter
-            # Simulate the following race condition:
-            # 1. The first 10 polls by MonitorsManager is such that DockerMonitor has not yet started. Therefore,
-            #     the docker version is None
-            # 2. After the 20th poll, docker version is set
-            # 3. After the 30th poll, docker mode changes to docker_api|raw
-            # 4. After the 40th poll, docker mode changes to docker_api|api
-            #
-            # Note: (3) and (4) do not happen in real life.  We force these config changes to test permutations
-            # of user agent fragments for different config scenarios
-            #
-            # Total number of times the user_agent_callback is called should be twice:
-            # - once for when docker version is None (fragment is 'docker=true')
-            # - once for when docker version changes to a real number
-            fake_docker_version = '18.09.2'
-            docker_mon = manager.monitors[0]
-            original_get_user_agent_fragment = docker_mon.get_user_agent_fragment
-            original_monitor_config_get = docker_mon._config.get
-
-            def fake_get_user_agent_fragment():
-                result = original_get_user_agent_fragment()
-                fragment_polls.increment()
-                return result
-
-            def fake_fetch_and_set_version():
-                # Simulate slow-to-start DockerMonitor where version is set only after 10th poll by MonitorsManager
-                # Thus, polls 0-9 return in version=None which ultimately translates to 'docker=true' fragment
-                docker_mon._DockerMonitor__version_lock.acquire()
-                try:
-                    if fragment_polls.count() < 10:
-                        docker_mon._DockerMonitor__version = None
-                    else:
-                        docker_mon._DockerMonitor__version = fake_docker_version
-                finally:
-                    docker_mon._DockerMonitor__version_lock.release()
-
-            def fake_monitor_config_get(key):
-                # Fake the return values from MonitorConfig.get in order to exercise different permutations of
-                # user_agent fragment.
-                if key == 'log_mode':
-                    if fragment_polls.count() < 20:
-                        return 'syslog'
-                    else:
-                        return 'docker_api'
-                elif key == 'docker_raw_logs':
-                    if fragment_polls.count() < 30:
-                        return True
-                    else:
-                        return False
+            return manager
```

Comment:
This large diff is merely unindenting since it's no longer with a context manager.

(Similarly for the other large diffs in unit tests)